src/apps/service/apps/agave/listings.clj

```diff
@@ -27,12 +54,14 @@
       nil)))
 
 (defn search-apps
-  [agave search-term params]
+  [agave search-term params admin?]
   (try+
    (-> (.searchApps agave search-term)
+       (add-app-listing-job-stats admin?)
        (sort-apps params {:default-sort-field "name"})
        (apply-offset params)
-       (apply-limit params))
+       (apply-limit params)
+       (format-app-listing-job-stats false))
```

Comment:
You're right, this should be `admin?`.
That also reminded me that I haven't added an `/admin/apps/` endpoint to use these admin search functions yet 😊